resource-watch/resource-watch
# Redirect and change the title after registration.

## Overview
This PR adds additional logic after registration. The title on sign in page changes to 'Thank you for registration. Please, confirm your email and sign in'.
It helps users to understand that they have registered.
## Testing instructions
Go to http://localhost:9000/sign-in
Click "Register" link
Fill the form.
Click the register button. 
You will redirect to sign-in page with the title 'Thank you for registration. Please, confirm your email and sign in'.
## [Pivotal task](https://www.pivotaltracker.com/n/projects/1374154/stories/171052028)

---

## Checklist before submitting
